### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by bigcat88**
task from @DaphneMuller

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,16 +1,41 @@
...
+/apps/cloud_federation_api          @vitormattos @mejo-
+/apps/comments                      @marcelklehr @ChristophWurst
+/apps/contactsinteraction           @kesselb @miaulalala @Pytal @GretaD @hamza221 @st3iny
+/apps/dashboard                     @julien-nc @juliushaertl
 /apps/dav/lib/CalDAV                @ChristophWurst @miaulalala @tcitworld
 /apps/dav/lib/CardDAV               @ChristophWurst @miaulalala @tcitworld
+/apps/encryption                    @come-nc @icewind1991
+/apps/federatedfilesharing          @icewind1991 @danxuliu
+/apps/federation                    @vitormattos @datenangebot
+/apps/files                         @nextcloud/files
```
Comment (nickvergessen) [inline .github/CODEOWNERS:14]: > This CODEOWNERS file contains errors …

Groups do not work

**Comment by bigcat88** [comment]
Probably someone should add write access to this repo for
https://github.com/orgs/nextcloud/teams

I do not know how to do that(

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,16 +1,41 @@
...
+/apps/files_external                @icewind1991 @artonge
+/apps/files_sharing                 @skjnldsv @come-nc
+/apps/files_trashbin                @ChristophWurst @icewind1991
+/apps/files_versions                @artonge @icewind1991
+/apps/oauth2                        @julien-nc @Pytal
+/apps/provisioning_api              @provokateurin @nickvergessen
+/apps/settings                      @ChristophWurst @JuliaKirschenheuter
+/apps/systemtags                    @Antreesy @marcelklehr
+/apps/theming                       @skjnldsv @juliushaertl
+/apps/twofactor_backupcodes         @st3iny @miaulalala @Pytal
```
Comment (nickvergessen) [inline .github/CODEOWNERS:24]: this is duplicated in line 33 with another user list.

**Review by nickvergessen — CHANGES_REQUESTED** [review]
As per discussion the CODEOWNER should be mapped to the `/appinfo/info.xml` for the apps.
Whether or not the owners fully subscribe to the apps path is up to them

File: .github/CODEOWNERS (hunk 1 above)
Comment (bigcat88) [reply .github/CODEOWNERS]: replaced with group members

**Comment by DaphneMuller** [comment]
hey @nickvergessen , shall we have a chat about it 1:1? I tasked Alexander with this. The idea is that community PR's get assigned faster to the right people. I don't see why what Alexander did is a problem for you, maybe you can rephrase or explain me?

**Comment by ChristophWurst** [comment]
For transparency please keep the discussion here

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -1,16 +1,40 @@
...
+/apps/cloud_federation_api/appinfo/info.xml   @vitormattos @mejo-
+/apps/comments/appinfo/info.xml               @marcelklehr @ChristophWurst
+/apps/contactsinteraction/appinfo/info.xml    @kesselb @miaulalala @Pytal @GretaD @hamza221 @st3iny
+/apps/dashboard/appinfo/info.xml              @julien-nc @juliushaertl
+/apps/dav/lib/CalDAV                          @ChristophWurst @miaulalala @tcitworld
+/apps/dav/lib/CardDAV                         @ChristophWurst @miaulalala @tcitworld
+/apps/encryption/appinfo/info.xml             @come-nc @icewind1991
+/apps/federatedfilesharing/appinfo/info.xml   @icewind1991 @danxuliu
+/apps/federation/appinfo/info.xml             @vitormattos @datenangebot
+/apps/files/appinfo/info.xml                  @skjnldsv @ChristophWurst @ArtificialOwl @come-nc @artonge @icewind1991 @szaimen @susnux @Fenn-CS
```
Comment (ChristophWurst) [inline .github/CODEOWNERS:14]:
```suggestion
/apps/files/appinfo/info.xml                  @skjnldsv @Pytal @ArtificialOwl @come-nc @artonge @icewind1991 @szaimen @susnux @Fenn-CS
```

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -1,16 +1,40 @@
...
+/apps/files_external/appinfo/info.xml         @icewind1991 @artonge
+/apps/files_sharing/appinfo/info.xml          @skjnldsv @come-nc
+/apps/files_trashbin/appinfo/info.xml         @ChristophWurst @icewind1991
+/apps/files_versions/appinfo/info.xml         @artonge @icewind1991
+/apps/oauth2/appinfo/info.xml                 @julien-nc @Pytal
+/apps/provisioning_api/appinfo/info.xml       @provokateurin @nickvergessen
+/apps/settings/appinfo/info.xml               @ChristophWurst @JuliaKirschenheuter
+/apps/systemtags/appinfo/info.xml             @Antreesy @marcelklehr
+/apps/theming/appinfo/info.xml                @skjnldsv @juliushaertl
+/apps/updatenotification/appinfo/info.xml     @ChristophWurst @JuliaKirschenheuter
```
Comment (ChristophWurst) [inline .github/CODEOWNERS:24]:
```suggestion
/apps/updatenotification/appinfo/info.xml     @Pytal @JuliaKirschenheuter
```

**Review by ChristophWurst — CHANGES_REQUESTED** [review]
@ChristophWurst vs @Pytal

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -1,16 +1,40 @@
...
+/apps/user_ldap/appinfo/info.xml              @come-nc @blizzz
+/apps/user_status/appinfo/info.xml            @Antreesy @nickvergessen
+/apps/weather_status/appinfo/info.xml         @julien-nc @juliushaertl
+/apps/workflowengine/appinfo/info.xml         @blizzz @juliushaertl
 
 # Two-Factor Authentication
 # https://github.com/nextcloud/wg-two-factor-authentication#members
-/apps/twofactor_backupcodes        @ChristophWurst @miaulalala @nickvergessen
-*/TwoFactorAuth/*                  @ChristophWurst @miaulalala @nickvergessen
```
Comment (ChristophWurst) [inline .github/CODEOWNERS:9]: I intentionally set myself as owner of this path pattern. Please leave this in place

Comment (bigcat88) [reply .github/CODEOWNERS:9]: so we can mark this as resolved?

File: .github/CODEOWNERS (hunk 1 above)
Comment (Pytal) [reply .github/CODEOWNERS]: Teams should work according to https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file?

File: .github/CODEOWNERS [hunk 6]
```diff
@@ -1,16 +1,40 @@
...
-/core/templates/twofactor*         @ChristophWurst @miaulalala @nickvergessen
+/apps/twofactor_backupcodes/appinfo/info.xml  @st3iny @miaulalala @ChristophWurst
```
Comment (nickvergessen) [inline .github/CODEOWNERS:32]:
```suggestion
/apps/twofactor_backupcodes/appinfo/info.xml  @st3iny @miaulalala @ChristophWurst
/apps/twofactor_backupcodes                   @ChristophWurst @miaulalala @nickvergessen
```

**Review by nickvergessen — APPROVED** [review]
Reordered the groups and added "section headlines"

**Review by ChristophWurst — CHANGES_REQUESTED** [review]
Only checked my own responsibilities but not any other mapping of gmbh employees to this code owners file

**Review by ChristophWurst — APPROVED** [review]
Please squash

Looks ok but didn't check every owner in detail

**Comment by nickvergessen** [comment]
> Please squash

Or we save the precious CI time since 5 jobs finished already.

**Comment by ChristophWurst** [comment]
Reabse with skipci

**Comment by bigcat88** [comment]
> Reabse with skipci

should I create a new PR with these changes and [skip ci] or someone else will do it?

**Pull request merged**